src/light.js

```diff
@@ -0,0 +1,8 @@
+let defaultStyle = {};
+let lowlight;
+
+import highlight from './highlight';
+
+lowlight = require('lowlight/lib/core');
```

Comment:
we can just import this instead of the let and reassignment since it's no longer conditionally assigned